src/components/Movie/MovieDetails.js

```diff
@@ -36,4 +37,4 @@ const mapDispatchToProps = {
     fetchMovieDetails: appActions.fetchMovieDetails
 }
 
-export default connect(null, mapDispatchToProps)(MovieDetails);
+export default withRouter(connect(null, mapDispatchToProps)(MovieDetails));

```

Comment:
Just one suggestion, for this you can use redux compose.

```javascript
import { compose } from 'redux'

export default compose(
    connect(null, mapDispatchToProps),
    withRouter
)(MovieDetails)
```

Reply:
Okay.